website/docs/guides/dremio-lakehouse.md

```diff
@@ -0,0 +1,193 @@
+hoverSnippet: Learn how to build a data lakehouse with dbt Core and Dremio Cloud
+# time_to_complete: '30 minutes' commenting out until we test
+platform: 'dbt-core'
+icon: 'guides'
+hide_table_of_contents: true
+tags: ['Dremio', 'dbt Core']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+This is a guide will demonstrate how to build a data lakehouse with dbt Core 1.5+ and Dremio Cloud. You can simplify and optimize your data infrastructure with dot’s robust transformation framework and Dremio’s open and easy data lakehouse. The integrated solution empowers companies to establish a strong data and analytics foundation, fostering self-service analytics and enhancing business insights while simplifying operations by eliminating the necessity to write complex ETL pipelines. 
+
+## Prerequisites
+
+* You must have a [Dremio Cloud](https://docs.dremio.com/cloud/) account.
+* You must have Python 3 installed.
+* You must have dbt Core v1.5 or newer [installed](/docs/core/installation). 
+* You must have the Dremio adatper 1.5.0 or newer [installed and configured](/docs/core/connect-data-platform/dremio-setup) for Dremio Cloud.
+* You must have basic working knowledge of Git and the command line interface (CLI).
+
+## Validate your environment 
+
+Validate your environment by running the following commands in your CLI and verifying the results in your CLI:
+
+```shell
+
+$ python3 --version
+Python 3.11.4 # Must be Python 3
+
+```
+
+```shell
+
+$ dbt --version
+Core:
+  - installed: 1.5.0 # Must be 1.5+
+  - latest:    1.6.3 - Update available!
+
+  Your version of dbt-core is out of date!
+  You can find instructions for upgrading here:
+  https://docs.getdbt.com/docs/installation
+
+Plugins:
+  - dremio: 1.5.0 - Up to date! # Must be 1.5+
+
+```
+
+## Getting started
+
+1. Clone the Dremio dbt Core sample project from the [github repo](https://github.com/dremio-brock/DremioDBTSample/tree/master/dremioSamples).
+
+2. Open the relation.py file in the Dremio adapter directory `$HOME/Library/Python/3.9/lib/python/site-packages/dbt/adapters/dremio/relation.py` in your IDE and locate lines 51 and 52.
+
+Update lines 51 and 52 if they don't have the following syntax:
+
+```python
+
+PATTERN = re.compile(r"""((?:[^."']|"[^"]*"|'[^']*')+)""")
+return ".".join(PATTERN.split(identifier)[1::2])
+
+```
+
+The complete selection should look like this:
+
+```python
+def quoted_by_component(self, identifier, componentName):
+        if componentName == ComponentName.Schema:
+            PATTERN = re.compile(r"""((?:[^."']|"[^"]*"|'[^']*')+)""")
+            return ".".join(PATTERN.split(identifier)[1::2])
+        else:
+            return self.quoted(identifier)
+
+```
+
+This is required because the plugin doesn’t support schema names in Dremio containing dots and spaces.
+
+## Build your pipeline
+
+1. Create a `profile.yml` file in the `$HOME/.dbt/profiles.yml` path and add the following configs:
```

Comment:
i think it's profiles.yml with an s, right? https://docs.getdbt.com/docs/core/connect-data-platform/profiles.yml

```suggestion
1. Create a `profiles.yml` file in the `$HOME/.dbt/profiles.yml` path and add the following configs:
```

Reply:
but not sure if dremio is diff

Reply:
Nope! Good catch